### DISCUSSION TITLE
Manage recurring tasks in governance

### DISCUSSION THREAD

**Discussion by eric-intuitem** [intuitem/ciso-assistant-community#1654] [Ideas]
**Problem statement**
I want to be able to plan recurring and non-recurring tasks. A task shall have:
- an owner, the person defining and following the tasks
- a list of contributors (users or groups), who are in charge of performing the task
- a due date (for a non-recurring task)
- a recurrence plan for due date, like in outlook, for recurring task
- an optional link with applied control(s) that are concerned by the task
- a status for each due date
- an eta date set by contributors for each due date.
…
The tasks shall be managed in the governance menu. I want a nice vizualisation of the tasks in time (one line per task). 

**Additional context**
Need a clear specification in data-model.md

**Comment by ab-smith** [comment]
Follow on #1625
